.env.lacchain

```diff
@@ -72,7 +72,7 @@ REACT_APP_EOS_BP_JSON_ON_CHAIN_CONTRACT=producerjson
 REACT_APP_EOS_BP_JSON_ON_CHAIN_TABLE=producerjson
 REACT_APP_EOS_BP_JSON_ON_CHAIN_SCOPE=producerjson
 REACT_APP_TOKEN_SYMBOL=
-REACT_APP_NETWORK_URL=[{"label":"EOS Mainnet","value":"https://mainnet.eosio.online"},{"label":"LACChain EOSIO","value":"https://lacchain.eosio.online"},{"label":"Jungle Testnet","value":"https://jungle.eosio.online"},{"label":"Proton Testnet","value":"https://proton-testnet.eosio.online"},{"label":"Telos Testnet","value":"https://telos-testnet.eosio.online"}]
+REACT_APP_NETWORK_URL=[{"label":"EOS Mainnet","value":"https://mainnet.eosio.online","mainnet":true,"pair":"eos","icon":"eos"},{"label":"LACChain EOSIO","value":"https://lacchain.eosio.online","mainnet":false,"pair":null,"icon":"lacchain"},{"label":"Proton","value":"https://proton.eosio.online","mainnet":true,"pair":"proton","icon":"proton"},{"label":"WAX","value":"https://wax.eosio.online","mainnet":true,"pair":"wax","icon":"wax"},{"label":"Telos","value":"https://telos-testnet.eosio.online","mainnet":true,"pair":"telos","icon":"telos"},{"label":"Jungle Testnet","value":"https://jungle.eosio.online","mainnet":false,"pair":"eos","icon":"jungle"},{"label":"Proton Testnet","value":"https://proton-testnet.eosio.online","mainnet":false,"pair":"proton","icon":"proton"},{"label":"WAX Testnet","value":"https://wax-testnet.eosio.online","mainnet":false,"pair":"wax","icon":"wax"},{"label":"Telos Testnet","value":"https://telos-testnet.eosio.online","mainnet":false,"pair":"telos","icon":"telos"}]
```

Comment:
Wrong URL for telos mainnet

`{"label":"Telos","value":"https://telos-testnet.eosio.online","mainnet":true,...`

should be : 

``{"label":"Telos","value":"https://telos.eosio.online","mainnet":true,...`